pre/remap_restart/remap_catchANDcn.py

```diff
@@ -225,6 +244,195 @@ def copy_merra2(self):
+            "default": "",
+        },  
+
+        {
+            "type": "select",
+            "name": "output:shared:bc_base",
+            "message": ("\nSelect " + message_bc_base_new),
+            "choices": choices_bc_base,
+            "default": get_default_bc_base(),
+        },
+
+        {
+            "type": "select",
+            "name": "output:shared:bc_version",
+            "message": message_bc_ops_new,
+            "choices": choices_bc_ops,
+            "default": "NL3",
+        },
+        {   
+            "type": "select",
+            "name": "output:shared:bc_version",
+            "message": message_bc_other_new,
+            "choices": choices_bc_other,
+            "when": lambda x:  x["output:shared:bc_version"] == 'Other',
+        },
+
+        {   
+            "type": "select",
+            "name": "output:surface:EASE_grid",
+            "message": "Select grid for new restart:\n",
+            "choices": ['EASEv2_M03', 'EASEv2_M09', 'EASEv2_M25', 'EASEv2_M36', 'Cubed-Sphere']
+        },
+
+        {
+            "type": "text",
+            "name": "output:shared:agrid",
+            "message": message_agrid_new,
+            "default": 'C360',
+            "validate": lambda text : text in validate_agrid,
+            "when": lambda x : x['output:surface:EASE_grid'] == 'Cubed-Sphere',
+        },
+
+        {
+            "type": "select",
+            "name": "output:shared:ogrid",
+            "message": message_ogrid_in,
+            "choices": choices_ogrid_data,
+            "default": lambda x: data_ocean_default(x.get('output:shared:agrid')),
+            "when": lambda x : x['output:surface:EASE_grid'] == 'Cubed-Sphere',
+        },
+
+        # dummy (invisible) question to remove parenthetical comments from selected output:shared:ogrid
+        {
+            "type": "text",
+            "name": "output:shared:ogrid",
+            "message": "remove the comment of ogrid",
+            # always return false, so questions never shows but changes ogrid
+            "when": lambda x: remove_ogrid_comment(x, 'OUT')
+        },
+
+
+        {
+            "type": "text",
+            "name": "slurm_pbs:qos",
+            "message": message_qos,
+            "default": "debug",
+        },
+
+        {
+            "type": "text",
+            "name": "slurm_pbs:account",
+            "message": message_account,
+            "default": get_account(),
+        },
+        {
+            "type": "text",
+            "name": "slurm_pbs:partition",
+            "message": message_partition,
+            "default": '',
+        },  
+
+   ]
+
+   answers = questionary.prompt(questions)   
+   yyyy = answers['input:shared:yyyymmddhh'][0:4]
+   mm   = answers['input:shared:yyyymmddhh'][4:6]
+   dd   = answers['input:shared:yyyymmddhh'][6:8]
+   hh   = answers['input:shared:yyyymmddhh'][8:10]
+  
+   rst_dir      = answers['input:shared:rst_dir']+'/rs/ens0000/Y'+yyyy +'/M'+mm+'/'
```

Comment:
Not sure why the path (incl "ens0000" is hardwired here again.  In any case, I suppose the "land_only" remapping is specific to the GEOSldas output directory structure.